config/config.yaml

```diff
@@ -117,6 +117,20 @@ datasets:
       # Relative path from the spras directory
       data_dir: "input"
 
+gold_standard:
+    -
+      label: gs0
```

Comment:
Do we need to copy the comment "# Labels can only contain letters, numbers, or underscores" as a reminder for the gold standard label?